AndroidNativeLoginTemplate/app/src/main/java/com/salesforce/androidnativelogintemplate/MainApplication.kt

```diff
@@ -142,14 +142,17 @@ class MainApplication : Application() {
         }
 
         /**
-         * Executes the Google reCAPTCHA client for a new login action token.
+         * Executes the Google reCAPTCHA client for a new token for the
+         * specified action.
+         * @param reCaptchaAction A Google reCAPTCHA SDK reCAPTCHA action
          * @param completion The function to invoke with the new token or null
          * if the token could not be obtained
          */
-        internal fun executeLoginAction(
+        internal fun executeReCaptchaAction(
+            reCaptchaAction: RecaptchaAction,
```

Comment:
Now the sample app demonstrates how to use appropriate reCAPTCHA actions as documented by Google.